### PR TITLE
Add documentation target

### PR DESCRIPTION
**Pull request by olivierapivideo**
- Add Documentation target
- Update CI to fail if at least one check in the generated clients fails

### CODE REVIEW

File: templates/documentation/.github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Run unit tests
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Test
+        run: exit 0
```
Comment (ThibaultBee) [inline templates/documentation/.github/workflows/test.yml:8]: Is this file only for debug?

File: templates/documentation/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+This repository contains the markdown files and the OpenAPI description used to generate the
+documentation of the api.video.
+
+The content of this repository is generated by the api.video API client generator.
+
+# FAQ
+
+If you have any questions, ask us in the [community](https://community.api.video). Or
+use [issues](https://github.com/apivideo/api.video-documentation/issues)..
```
Comment (ThibaultBee) [inline templates/documentation/README.md:27]: Replace `..` with `.`

File: templates/documentation/.github/workflows/test.yml (hunk 1 above)
Comment (olivierapivideo) [reply templates/documentation/.github/workflows/test.yml:8]: all target repos are supposed to have tests, for now documentation has tests that are always happy

Comment (ThibaultBee) [reply templates/documentation/.github/workflows/test.yml:8]: Could you add a comment about this?

Comment (olivierapivideo) [reply templates/documentation/.github/workflows/test.yml:9]: done

**Pull request merged**